source/jormungandr/tests/journey_common_tests.py

```diff
@@ -346,24 +346,6 @@ def test_journeys_date_valid_not_zeropadded(self):
         assert 'message' in response
         assert "Unable to parse datetime, year" in response['message']
 
-    def test_journeys_do_not_loose_precision(self):
-        """do we have a good precision given back in the id"""
```

Comment:
Why was this test added when it was? Because it was clearly added for a purpose?